helpers/pretests_check.rb

```diff
@@ -52,4 +54,12 @@ def self.request_to(path)
   rescue StandardError
     false
   end
+
+  def self.palladium_token?
+    status = false
+    status = true if File.file?("#{ENV['HOME']}/.palladium/token")
+    status = true unless ENV['PALLADIUM_TOKEN']
```

Comment:
This return true if `ENV['PALLADIUM_TOKEN']` not defined

Correct line is 
`status = true if ENV['PALLADIUM_TOKEN']`